server/src/main/java/org/elasticsearch/index/refresh/RefreshStats.java

```diff
@@ -129,6 +160,27 @@ public int getListeners() {
         return listeners;
     }
 
+    /**
+     * Timestamp of the last refresh.
+     */
+    public long getLastRefreshTime() {
```

Comment:
I decided to make these values timestamps rather than a duration (like `timeSinceLastRefresh`) because from what I could tell timestamps are more widely used in the stats, but also because it was proving it be very difficult to write solid tests when the stat grows with time.